Package@swift-5.9.swift

```diff
@@ -0,0 +1,49 @@
+// swift-tools-version:5.9
+import PackageDescription
+
+let swiftSettings: [SwiftSetting] = [
+    .enableUpcomingFeature("ExistentialAny"),
+    .enableUpcomingFeature("ConciseMagicFile"),
+    .enableUpcomingFeature("ForwardTrailingClosures"),
+    .enableUpcomingFeature("DisableOutwardActorInference"),
+    .enableExperimentalFeature("StrictConcurrency=complete"),
+]
```

Comment:
It recently came to my attention that you can define this variable at the end of the file, but it needs to be computed instead of stored.

I'm not sure if you want to keep `ExistentialAny` since it's going to be removed. As i've mentioned before i do like it.

I'd also use `StrictConcurrency` instead of `StrictConcurrency=complete` since Holly removed the option to choose `targeted` from future Swift versions (she added a new `upcomingFeature` flag for it, instead of `experimental`, but it'snot available in 5.9)

Reply:
I prefer `ExistentialAny` syntax. It's not going to be removed, it's just not going to become "required" until Swift 7 (according to [Features.def](https://github.com/apple/swift/blob/main/include/swift/Basic/Features.def#L130).

As for `StrictConcurrency`, it does no harm to specify `complete` for now. When I can switch to the `upcoming` version, I'll stop bothering.